### PR TITLE
Test run fixes - WIP

### PR DESCRIPTION
**Pull request by andymck**
IGNORE

this is a WIP branch, where i am pushing fixes from issues encountered during verifier testing on mainnet

This fixes two issues observed during ongoing testing on mainnet:

1.  The incoming timestamps from ingestor are in nano secs, verifier was expecting seconds.  
2. Whilst verifier was iterating over a list of beacons to process, should one of those fail to verify, verifier would halt processing of all beacons at that point rather than moving to the next beacon iteration.

### CODE REVIEW

**Review by bbalser — APPROVED** [review]

File: poc_iot_verifier/src/loader.rs [hunk 1]
```diff
@@ -184,7 +184,7 @@ impl Loader {
                     id_hash,
                     packet_data,
                     buf_as_vec,
-                    &datetime_from_epoch(event.timestamp),
+                    &event.timestamp.to_timestamp_nanos()?,
```
Comment (jeffgrunewald) [inline poc_iot_verifier/src/loader.rs:187]: why are we doing witness ingest report to nanos but beacon ingest to millis?

File: poc_iot_verifier/src/loader.rs [hunk 2]
```diff
@@ -196,7 +196,7 @@ impl Loader {
                     &self.pool,
                     &id,
                     &event.data,
-                    &datetime_from_epoch(event.timestamp),
+                    &event.timestamp.to_timestamp()?,
```
Comment (jeffgrunewald) [inline poc_iot_verifier/src/loader.rs:199]: and here we're just going to the default timestamp units? shouldn't these all be to the same level of precision (presumably millis or nanos)?

File: poc_iot_verifier/src/loader.rs (hunk 1 above)
Comment (andymck) [reply poc_iot_verifier/src/loader.rs:187]: this branch is stale, fixed in latest

File: poc_iot_verifier/src/loader.rs (hunk 2 above)
Comment (andymck) [reply poc_iot_verifier/src/loader.rs:199]: as above, branch is stale, fixed in latest

**Pull request closed without merging**